README.md

```diff
@@ -1,12 +1,14 @@
 # stream-go
 
-Master Branch: [![Build Status](https://travis-ci.org/GetStream/stream-go.svg?branch=master)](https://travis-ci.org/GetStream/stream-go)
-Dev Branch: [![Build Status](https://travis-ci.org/GetStream/stream-go.svg?branch=dev)](https://travis-ci.org/GetStream/stream-go)
+[stream-go](https://github.com/GetStream/stream-go) is a (beta) Go client for [Stream](https://getstream.io/).
 
-[![godoc](https://godoc.org/github.com/GetStream/stream-go?status.svg)](https://godoc.org/github.com/GetStream/stream-go)
-[![codecov](https://codecov.io/gh/GetStream/stream-go/branch/master/graph/badge.svg)](https://codecov.io/gh/GetStream/stream-go)
+You can sign up for a Stream account at https://getstream.io/get_started.
```

Comment:
I just had a quick look in GA and we can already drill down to see the Referrer header from individual GH projects. Though I guess you mean UTM tags in the actual URL?